### PR TITLE
trustar to trustar2

### PR DESCRIPTION
**Pull request by YaPeL**
*WHAT*: rename trustar to trustar2

*WHY*: so it doesn't collied with the old sdk

*HOW*: I typed the number 2 at the end of trustar

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
     packages=find_packages(exclude=("tests",)),
     version=version,
     author='TruSTAR Technology, Inc.',
-    author_email='support@trustar.co',
+    author_email='support@trustar2.co',
```
Comment (jagrvargen) [inline setup.py:15]: does this need to change?

File: trustar2/log.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Seems like from time to time people do a from trustar import *, lets minimize the chances of something ugly happening
+# Seems like from time to time people do a from trustar22 import *, lets minimize the chances of something ugly happening
```
Comment (jagrvargen) [inline trustar2/log.py:1]: haha uhhhh

**Review by marcos-vivar — APPROVED** [review]
LGTM! :shipit:

File: trustar2/log.py (hunk 2 above)
Comment (jagrvargen) [reply trustar2/log.py:1]: is it trustar22?

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -23,6 +23,6 @@ repos:
     hooks:
     -   id: tests
         name: Tests
-        entry: sh -c "PYTHONPATH=${PYTHONPATH}:${PWD} pytest --cov=trustar"
+        entry: sh -c "PYTHONPATH=${PYTHONPATH}:${PWD} pytest --cov=trustar22"
```
Comment (jagrvargen) [inline .pre-commit-config.yaml:26]: here too

File: trustar2/log.py (hunk 2 above)
Comment (YaPeL) [reply trustar2/log.py]: sorry, I got this code from my time machine and I forgot to delete this part (?) Fixed now

File: .pre-commit-config.yaml (hunk 3 above)
Comment (YaPeL) [reply .pre-commit-config.yaml]: done

**Review by jagrvargen — APPROVED** [review]
lgtm!

**Pull request merged**